From 35240dd3d3869f84b0e7f3ce76760c5e9cdabc1a Mon Sep 17 00:00:00 2001
From: Marko Man <darkobas@gmail.com>
Date: Fri, 1 Dec 2017 14:06:10 -0500
Subject: [PATCH 51/90] build: recovery: add vendor to exclusion list

on some devices there is a vendor symlink in rootdir.
Ignore that for recovery.

Change-Id: Id0e54f7d135418cbe3adc1985c2dd26e260ab495
---
 core/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/Makefile b/core/Makefile
index 6b5f6d74c..ca3fed39e 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1456,7 +1456,7 @@ define build-recoveryramdisk
   $(hide) mkdir -p $(TARGET_RECOVERY_ROOT_OUT)/etc $(TARGET_RECOVERY_ROOT_OUT)/sdcard $(TARGET_RECOVERY_ROOT_OUT)/tmp
   # Copying baseline ramdisk...
   # Use rsync because "cp -Rf" fails to overwrite broken symlinks on Mac.
-  $(hide) rsync -a --exclude=etc --exclude=sdcard $(IGNORE_RECOVERY_SEPOLICY) $(IGNORE_CACHE_LINK) $(TARGET_ROOT_OUT) $(TARGET_RECOVERY_OUT)
+  $(hide) rsync -a --exclude=etc --exclude=sdcard --exclude=vendor $(IGNORE_RECOVERY_SEPOLICY) $(IGNORE_CACHE_LINK) $(TARGET_ROOT_OUT) $(TARGET_RECOVERY_OUT)
   # Modifying ramdisk contents...
   $(if $(BOARD_RECOVERY_KERNEL_MODULES), \
     $(call build-image-kernel-modules,$(BOARD_RECOVERY_KERNEL_MODULES),$(TARGET_RECOVERY_ROOT_OUT),,$(call intermediates-dir-for,PACKAGING,depmod_recovery)))
-- 
2.17.0

